app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantItemViewModelFactory.kt

```diff
@@ -0,0 +1,13 @@
+package com.google.samples.apps.sunflower.viewmodels
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.google.samples.apps.sunflower.data.Plant
+
+class PlantItemViewModelFactory(private val plant: Plant) : ViewModelProvider.NewInstanceFactory() {
```

Comment:
For item of list, really find unnecessary.
Up 2 U. :smile_cat:

Reply:
Same as commented above:
> Yes, it seems that having a factory for the view model of the item of a list is a bit useless. I have updated the code for removing the factoring and instantiating the view model directly